proto/batch_plan.proto

```diff
@@ -83,6 +83,14 @@ message InsertNode {
   // An optional field and will be `None` for tables without user-defined pk.
   // The `BatchInsertExecutor` should add a column with NULL value which will
   // be filled in streaming.
+  message DefaultColumns {
```

Comment:
Do you think it's necessary to move it out of the `InsertNode` scope? I guess it's required by the table catalog as well.